### PR TITLE
TDI-43747 - Update Jython

### PR DESCRIPTION
**Pull request by coheigea**
This task is to update Jython to fix a high level CVE reported by Veracode.

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/252/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/252/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3ATDI-43747)

File: components/components-processing/processing-runtime/pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>org.python</groupId>
             <artifactId>jython-standalone</artifactId>
-            <version>2.7.0</version>
+            <version>2.7.1b3</version>
```
[the remark this answers is not in the document]
Comment (coheigea) [reply components/components-processing/processing-runtime/pom.xml]: @Voievidko Yes good point, I'll update the PR.

[the remark this answers is not in the document]
Comment (Voievidko) [reply components/components-processing/processing-runtime/pom.xml]: Thanks Colm!

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/290/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/290/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3ATDI-43747)

**Pull request merged**